org.qutebrowser.qutebrowser.json

```diff
@@ -0,0 +1,217 @@
+{
+    "app-id": "org.qutebrowser.qutebrowser",
+    "branch": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "qutebrowser-wrapper",
+    "finish-args": [
+        /* Nested sandboxes are currently not support by flatpak */
+        "--env=QTWEBENGINE_DISABLE_SANDBOX=1",
+        /* Allow direct rendering to use OpenGL and access to webcams */
+        "--device=all",
+        /* Allow to read all files from the home directory */
+        "--filesystem=home:ro",
+        /* Allow to save downloads */
+        "--filesystem=xdg-download",
+        /* Allow IPC access for performance reasons (X shared memory extension) */
+        "--share=ipc",
+        /* Allow network access */
+        "--share=network",
+        /* Allow to play sounds using pulseaudio */
+        "--socket=pulseaudio",
+        /* Allow to provide a graphical UI */
+        "--socket=wayland",
+        "--socket=x11"
+    ],
+    "rename-desktop-file": "qutebrowser.desktop",
+    "rename-appdata-file": "qutebrowser.appdata.xml",
+    "rename-icon": "qutebrowser",
+    "cleanup": [
+        "/bin/a2x",
+        "/bin/a2x.py",
+        "/bin/asciidoc",
+        "/bin/asciidoc.py",
+        "/bin/pylupdate5",
+        "/bin/pyrcc5",
+        "/bin/pyuic5",
+        "/bin/sip",
+        "/etc/asciidoc",
+        "/include",
+        "/lib/debug",
+        "/share/man/man1/asciidoc.1"
+    ],
+    "modules": [
+        "shared-modules/udev/udev-175.json",
+        {
+            "name": "PyQt5",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=/app PyQt5-5.10.1-5.10.1-cp35.cp36.cp37.cp38-abi3-manylinux1_x86_64.whl"
```

Comment:
If possible avoid binary `whl` files. It may happen to be ABI compatible but its just easier to reason about everything when its built.

Reply:
Thanks for having a look at the merge request. Unfortunately the performance of QTWebEngine in the KDE platform is quite bad. But I haven't yet reported the issues upstreams.

In the long run I would like to rebase the flatpak on the KDE platform.

Reply:
I think we should fix these before this gets added not after. The KDE runtime folks are very active and helpful.